e2e/cypress/integration/enterprise/saml/okta_login_spec.js

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import users from '../../../fixtures/saml_users.json';
+
+//assumes that the SAML certificates+keys are already present in the config folder
+//assumes that Cypress.env.json is copied in the same folder with Cypress.json
+//assumes thet CYPRESS_* variables are set
+//In the Okta Applicaions->Okta MM App define attribute statements:
+//Name: UserType -> Value: user.userType
+//Name: IsAdmin -> Value: user.isAdmin
+//Name: IsGuest -> Value: user.isGuest
+//In the Okta Profile Editor, add following custom types:
+//- for Okta app: variablename: isAdmin(boolean, isGuest(boolean) (userType is already defined)
+//- for Okta MM app: variablename: UserType(string),IsGuest(boolean), IsAdmin(boolean)
```

Comment:
I wanted them to be documented somewhere and not be "lost", since they are not part of the saml-okta-setup.txt document - in case we have to redo this later. I will add them there instead to consolidate steps - though that makes them less discoverable to the MM developer.

Reply:
Yes, but we have the setup steps in multiple places and it might be difficult to maintain (like one is updated & the other is not). Moreover we need to download the files from the platform-private repo anyways. Hence its better to have it in a single place.